src/style.less

```diff
@@ -1,3 +1,11 @@
+.main-container {
```

Comment:
Would it work to have these styles apply to `.rootContainer` instead? The containing div element won't have the `.main-container` class when used in Bluegenes.